source/documentation/administration_guide/topics/Grafana_dashboards.adoc

```diff
@@ -46,6 +46,12 @@ The following dashboards are available in the initial Grafana setup to report Da
 |===
 
 
+[NOTE]
+====
+The Grafana dashboards includes direct links to the Red Hat Virtualization Administration Portal, allowing you to quickly view additional details for your clusters, hosts, and virtual machines. 
```

Comment:
Just Making sure we use 'Red Hat Virtualization Administration Portal' on downstream, and 'oVirt Virtualization Administration Portal' on upstream.